content/en/docs/setup/getting-started/index.md

```diff
@@ -68,146 +52,333 @@ Download the Istio release which includes installation files, samples, and the
+[Istio Ingress Gateway](/docs/concepts/traffic-management/#gateways), which maps a path to a
+route at the edge of your mesh.
+
+1.  Connect this application to be published by the Istio gateway:
 
     {{< text bash >}}
-    $ kubectl get pods -n istio-system
-    NAME                                    READY   STATUS    RESTARTS   AGE
-    grafana-5cc7f86765-lqdzk                1/1     Running   0          2m25s
-    istio-egressgateway-97445fd58-vx9s9     1/1     Running   0          2m29s
-    istio-ingressgateway-565b5cd49f-9knzt   1/1     Running   0          2m29s
-    istio-tracing-8584b4d7f9-ft88q          1/1     Running   0          2m25s
-    istiod-6476b4cb9b-8gfj2                 1/1     Running   0          2m41s
-    kiali-8559969566-wm62d                  1/1     Running   0          2m24s
-    prometheus-965cff5ff-pprnh              2/2     Running   0          2m24s
+    $ kubectl apply -f @samples/bookinfo/networking/bookinfo-gateway.yaml@
+    gateway.networking.istio.io/bookinfo-gateway created
+    virtualservice.networking.istio.io/bookinfo created
     {{< /text >}}
 
-## Enable automatic sidecar injection {#enable-injection}
+1.  Confirm the gateway has been created:
+
+    {{< text bash >}}
+    $ kubectl get gateway
+    NAME               AGE
+    bookinfo-gateway   32s
+    {{< /text >}}
+
+### Determining the ingress IP and ports
+
+Follow these instructions to set the `INGRESS_HOST` and `INGRESS_PORT` variables
+for accessing the gateway. Use the tabs to choose the instructions for your
+chosen platform:
+
+{{< tabset category-name="gateway-ip" >}}
+
+{{< tab name="Minikube" category-value="external-lb" >}}
+
+Set the ingress ports:
+
+{{< text bash >}}
+$ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
+$ export SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="https")].nodePort}')
+{{< /text >}}
+
+Ensure a port was successfully assigned to each environment variable:
+
+{{< text bash >}}
+$ echo $INGRESS_PORT
+32194
+{{< /text >}}
+
+{{< text bash >}}
+$ echo $SECURE_INGRESS_PORT
+31632
+{{< /text >}}
+
+Set the ingress IP:
+
+{{< text bash >}}
+$ export INGRESS_HOST=$(minikube ip)
+{{< /text >}}
+
+Ensure an IP address was successfully assigned to the environment variable:
+
+{{< text bash >}}
+$ echo $INGRESS_HOST
+192.168.4.102
+{{< /text >}}
 
-A benefit of Istio is automatic sidecar injection, which allows your applications to work in the service mesh without modification. To take advantage of this feature, enable sidecar injection by adding the `istio-injection=enabled` label to the Kubernetes namespaces in which you plan to deploy your applications. For more information, see [Istio sidecar injector](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection).
+Run this command in a new terminal window to start a Minikube tunnel that
```

Comment:
I'm still not sure this, the `minikube tunnel` is needed. It probably doesn't hurt.

Reply:
I had to add it for my environment to work. It literally was dead for hours until specifically this step.